mitodl/open-discussions
# Added API support for sorting comments

**Blocked on #429 because it is based off that PR**

#### What are the relevant tickets?
Fixes #193 

#### What's this PR do?
Adds API support for sorting comments

#### How should this be manually tested?
Hit the API for comments on a post and append `?sort=new|old|best` to the end of the url and verify you see results. No query string should return the same results as "best".

Note: you need a decent variety of posts/comments/users to make these sorts sufficiently different

#### What GIF best describes this PR or how it makes you feel?
![Sorting](https://media.giphy.com/media/Tl2AK8HOHj7SU/giphy.gif)
